test/components/time_ago_component_test.js

```diff
@@ -0,0 +1,12 @@
+import {expect} from 'chai'
+import {componentTest} from './helpers.js'
+
+componentTest('time_ago_component', it => {
+
+  it('works', async page => {
+    await expect(page).to.have.selector('body')
+    const el = await page.$('body')
+    await expect(el).to.be.visible()
+  })
+
+})
```

Comment:
does this file do something? 
I couldn't find `./helpers.js` and don't see `chai` as a dependency

Reply:
Oops this was from an old iteration of the tests!